packages/core/lottie-react-native.podspec

```diff
@@ -23,7 +23,7 @@ Pod::Spec.new do |s|
   s.author                  = package["author"]
   s.homepage                = package["homepage"]
 
-  s.ios.deployment_target   = '12.4'
+  s.ios.deployment_target   = '13.0'
```

Comment:
Was really hoping to avoid this until another major version of RN came out, but i guess our hand is being forced. Can you set this to be 13.4 instead of 13 to match RN 73's iOS requirements?